index.html

```diff
@@ -16,7 +16,7 @@
       </ul>
       
     </nav>
```

Comment:
Sorry if I did not explain well in the previous review. 
This is an Example of an inline comment.
- [ ] Please kindly remove empty unnecessary empty lines of code. It's always best practice to keep your code clean and professional.

Reply:
Ok understand

Reply:
How can I do that? 
The video starts in the middle.

Reply:
Done remove any empty unnecessary empty

Reply:
- [ ] **REQUIRED CHANGES**

- [ ] Please, rephrase your review as par how the previous reviewer suggested. Your review sounds hash and this is a red flag while communicating with someone, in turn, will make your review process not a good experience.